src/parse_file.py

```diff
@@ -27,9 +27,9 @@ def import_from_xrdml(self, file):
     scan_type = content.getElementsByTagName("scan")
     scan_axis = scan_type[0].attributes["scanAxis"].value
     if scan_axis.startswith("2Theta"):
-        scan_axis = _("2Theta")
+        scan_axis = "2Theta"
```

Comment:
These shouldn't be translated, as they are used later on in the code to parse the xml. Perhaps will add a translatable version specifically for the label later in a seperate PR.